Core.DomainModel/GDPR/Read/DataProcessingRegistrationReadModel.cs

```diff
@@ -60,6 +60,8 @@ public DataProcessingRegistrationReadModel()
         public YesNoUndecidedOption? IsOversightCompleted { get; set; }
 
         public DateTime? OversightScheduledInspectionDate { get; set; }
+        public int? MainContractId { get; set; }
+        public bool MainContractIsActive { get; set; }
```

Comment:
ActiveAccordingToMainContract because it may be active even if there is no contract associated.. just like in itsystem

Reply:
also add a combined active state... check the other read models for naming